### ISSUE TITLE
Returning gives columns in undefined order

### ISSUE THREAD

**Issue by nicolasparada** [ulule/loukoum#83]
Column order while using `RETURNING` is undefined.
Given this as example:

```go
package main

import (
	"fmt"

	lk "github.com/ulule/loukoum/v3"
)

func main() {
	query, args := lk.
		Insert("tests").
		Set(lk.Pair("value", "test")).
		Returning("id, value, created_at, updated_at, deleted_at").
		Query()
	fmt.Printf("%q %+v\n", query, args)
}
```

It gives:

```
"INSERT INTO tests (value) VALUES ($1) RETURNING created_at, deleted_at, id, updated_at, value" [test]
```

Expected:

```
"INSERT INTO tests (value) VALUES ($1) RETURNING id, value, created_at, updated_at, deleted_at" [test]
```

---

_Version: 3.2.1_

**Comment by nicolasparada** [comment]
Well, looks like it's in alphabetical order I guess 🙄

**Comment by novln** [comment]
Yes, it is in alphabetical order :slightly_smiling_face: 
Do you have another question?

**Comment by nicolasparada** [comment]
It's not a question, but about the expected behaviour while using `.Query()`.
Maybe the order doesn't matter while using the other "scan" mechanism loukoum provides.
But when building the query, one expect it to be in the same order provided.

**Comment by novln** [comment]
Yeah you are right, it's a leftover when it was only possible to use loukoum with `sqlx`.
The order shouldn't change for this statement.

This behaviour is executed here: https://github.com/ulule/loukoum/blob/master/stmt/returning.go#L27-L30

Don't hesitate to submit a pull request. Otherwise, I'll try to fix that when I have some free time.

Thank you for this report.

**Comment by novln** [comment]
This is fixed with the new version `v3.3.0`.
